cmd/server/cmd/test_server.go

```diff
@@ -41,7 +41,7 @@ var defaultSeverArgs = serverArgs{
 	configFetchIntervalSec:        3,
 	configIdentityAttribute:       "target.service",
 	configIdentityAttributeDomain: "",
-	useAst: false,
+	useAst: true,
```

Comment:
I think for now this should be a paramter.. mainly because we want more test that will be written in next 1-2 using your framework to by default exercise the IL code.. and find bugs similar to what you discovered.. so, let's make this false by default and let each test override it explicitly.

Reply:
Once IL is the default, the flag goes away and we will clear the test then